examples/Cip30.purs

```diff
@@ -0,0 +1,114 @@
+-- | This module demonstrates the use of the CIP-30 functions
+-- | using an external wallet.
+module Ctl.Examples.Cip30
+  ( main
+  , example
+  , contract
+  , noSignDataContract
+  ) where
+
+import Contract.Prelude
+
+import Contract.Config (ConfigParams, testnetNamiConfig)
+import Contract.Log (logInfo')
+import Contract.Monad (Contract, launchAff_, liftContractAffM, runContract)
+import Contract.Prim.ByteArray (rawBytesFromAscii)
+import Contract.Wallet
+  ( WalletExtension
+  , apiVersion
+  , getChangeAddress
+  , getNetworkId
+  , getRewardAddresses
+  , getUnusedAddresses
+  , getWallet
+  , icon
+  , isEnabled
+  , isWalletAvailable
+  , name
+  , signData
+  , walletToWalletExtension
+  )
+import Control.Monad.Error.Class (liftMaybe)
+import Data.Array (head)
+import Effect.Exception (error)
+
+main :: Effect Unit
+main = example testnetNamiConfig
+
+example :: ConfigParams () -> Effect Unit
+example cfg = launchAff_ do
+  mWallet <- runContract cfg getWallet
+  let mSupportWallet = walletToWalletExtension =<< mWallet
+  _ <- traverse nonConfigFunctions mSupportWallet
+  runContract cfg contract
+
+nonConfigFunctions :: WalletExtension -> Aff Unit
+nonConfigFunctions extensionWallet = do
+  log "Functions that don't depend on `Contract`"
+  performAndLog "isWalletAvailable" (liftEffect <<< isWalletAvailable)
+  performAndLog "isEnabled" isEnabled
+  performAndLog "apiVersion" apiVersion
+  performAndLog "name" name
+  performAndLog "icon" icon
+  where
+  performAndLog
+    :: forall (a :: Type)
+     . Show a
+    => String
+    -> (WalletExtension -> Aff a)
+    -> Aff Unit
+  performAndLog msg f = do
+    result <- f extensionWallet
+    log $ msg <> ":" <> (show result)
+
+-- a version without the `signData` call, which is not implemented for KeyWallet
```

Comment:
`signData` has been implemented, so the comment should be deleted and the function renamed accordingly.